doc/stdenv/platform-notes.chapter.md

```diff
@@ -140,81 +118,61 @@ Some common issues when packaging software for Darwin:
   }
   ```
 
-  If your derivation has multiple versions of this hook in its inputs (e.g., because it has been propagated by one of your dependencies), it will use the highest deployment target available.
+Note: It is possible to have multiple, different instances of `darwinMinVerisonHook` in your inputs. When that happens, the one with the  highest version is always used.
 
-  ```nix
-  stdenv.mkDerivation {
-    name = "libfoo-1.2.3"; # Upstream specifies the minimum supported version as 10.15.
-    buildInputs = [ qt6.qtbase (darwinMinVersionHook "10.15") ];
-  }
-  # Qt 6 specifies a minimum version of 12.0, so the minimum version resolves to 12.0.
-  ```
+##### Propagating a `darwinMinVersionHook`
 
+This follows the same guidelines as for propagating an SDK. In general, don’t.
 
-- You should rely on the default SDK when possible. If a package specifies a required SDK version, use that version (e.g., libuv requires 11.0, so it should use `apple-sdk_11`). When a package supports multiple SDKs, determine which SDK package to use based on the following rules of thumb:
+#### Picking an SDK version
```

Comment:
I think this can go above to address my earlier concern and because it’s a lot more common than specifying a minimum version or propagating anything.

Reply:
It now comes after the SDK and minimum version sections. Propagation is down at the very end of troubleshooting now.